### ISSUE TITLE
Tests not running on pull requests

### ISSUE THREAD

**Issue by quentinlintz** [quentinlintz/cmdtop#2]
I noticed as part of #1 automatic tests aren't running. This is probably because they're left out of the test.yml config.

Something like this may solve that problem:

```
on:
  push:
    branches: [main]
  pull_request:
    branches: [main]
```

**Comment by Ilhan-Personal** [comment]
hey @quentinlintz can i please be assigned this issue?

**Comment by quentinlintz** [comment]
Fixed by @SYSHIL in #3
